app/config/routing_test.yml

```diff
@@ -0,0 +1,12 @@
+app:
+  # The main bundle is PrestaShopCoreBundle which will load other dependencies.
+  resource: "@PrestaShopBundle/Resources/config/routing.yml"
+
+test:
+  resource: "../../tests/Resources/config/routes.yml"
+
+app_modules:
+  # Declare routing.yml file in modules/module-name/config folder.
+  # v1: only YAML format is supported for now.
+  resource: .
```

Comment:
I'm not sure to understand what this targets? It loads resource from the current folder? So from the `app/config` folder?

Reply:
This is a config related to module routes, which I took from the basic routing.yml. It appears that the resource is an unsupported value, see https://github.com/PrestaShop/PrestaShop/commit/d34787c489fedfa7bb251433dfe5236976fccc64